### PR TITLE
introduce mvn wrapper / mvnw

### PR DESCRIPTION
**Pull request by grafjo**

### CODE REVIEW

**Review by strieflin — CHANGES_REQUESTED** [review]
See https://github.com/carbynestack/amphora/pull/22#discussion_r827087715.

**Comment by grafjo** [comment]
> See [carbynestack/amphora#22 (comment)](https://github.com/carbynestack/amphora/pull/22#discussion_r827087715).

requested changes are done

File: 3RD-PARTY-LICENSES/sbom.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<attributionReport>
```
[the remark this answers is not in the document]
Comment (strieflin) [reply 3RD-PARTY-LICENSES/sbom.xml:1]: Discussed this with @sbckr. The `sbom.xml` should contain only things that are shipped in binary form (e.g. jar or docker image). The `maven-license-plugin` is not executed for the top-level `castor` module as it is `pom`-packaged. @grafjo: Please remove the `sbom.xml` file completely. The other files in the `3RD-PARTY-LICENSES` folder should be kept as we have to still fulfill the license obligations for the maven wrapper scripts.

**Review by strieflin — CHANGES_REQUESTED** [review]

File: 3RD-PARTY-LICENSES/sbom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Lila84) [reply 3RD-PARTY-LICENSES/sbom.xml]: sbom.xml is removed

**Review by strieflin — APPROVED** [review]
LGTM

**Pull request merged**